.github/workflows/build_linux_jax_wheels.yml

```diff
@@ -135,21 +154,33 @@ jobs:
             --therock-path="${{ inputs.tar_url }}" \
             dist_wheels
 
+      - name: Extract JAX version
+        id: extract_jax_version
+        run: |
+            # Extract JAX version from requirements.txt (e.g., "jax==0.8.0")
+            # Remove all whitespace from requirements.txt to simplify parsing
+            # Search for lines starting with "jax==" or "jaxlib==" followed by version (excluding comments)
+            # Extract the version number by splitting on '=' and taking the 3rd field
+            # [^#]+ matches one or more characters that are NOT '#', ensuring we stop before any inline comments
+            JAX_VERSION=$(tr -d ' ' < jax/build/requirements.txt \
+            | grep -E '^(jax|jaxlib)==[^#]+' | head -n1 | cut -d'=' -f3)
+            echo "jax_version=$JAX_VERSION" >> "$GITHUB_OUTPUT"
```

Comment:
We should move this into a script. See https://github.com/ROCm/TheRock/blob/main/docs/development/style_guide.md#prefer-python-scripts-over-inline-bash